auv_control.orogen

```diff
@@ -6,6 +6,7 @@ import_types_from "6dControl.hpp"
 using_library "motor_controller"
 
 typekit.export_types '/motor_controller/PIDSettings'
+typekit.export_types '/base/LinearAngular6DCommand'
```

Comment:
Well, in my opinion `import_types_from "base"` is practically a default orogen statement, meaning that whoever imported from auv_control, would probably import from base as well in order to have access to other basic types.

In go for `removing` this export. Any complaints?